src/components/dropdown/DropdownElement.tsx

```diff
@@ -21,7 +21,7 @@ export function DropdownElement(props: Props) {
     return (
       <Link
         to={props.to}
-        className="w-full text-left hover:bg-gray-100 z-50 block px-4 py-2 text-sm text-gray-700"
+        className={`w-full text-left hover:bg-gray-100 z-50 block px-4 py-2 text-sm text-gray-700 ${props?.className}`}
```

Comment:
Why props is marked optional? Parameter signature makes it required, I am not sure if we need this optional check?

Reply:
> Why props is marked optional? Parameter signature makes it required, I am not sure if we need this optional check?

Yes, sure, tbh that is one of my habits through programming :D. I set it on pretty much all places to make sure that app will not broke in some specific scenario or after some updates. I removed it.